llm_bench/python/requirements.txt

```
@@ -6,7 +6,8 @@ openvino-tokenizers
 openvino_genai
 auto-gptq>=0.5.1 # for gptq
 pillow
-torch
+torch==2.3
+torchvision==0.18
```

```suggestion
torch<2.4.0
torchvision<0.19.0
```
Will add requirements_2024.3.txt with all validated fixed versions like https://github.com/openvinotoolkit/openvino.genai/blob/releases/2024/2/llm_bench/python/requirements_2024.2.txt

---

Yeah, but I don't know how to force internal validation to use that file. It was simpler just to freeze here and trigger the validation from branch.